axes/tests/test_apps.py

```diff
@@ -0,0 +1,25 @@
+import warnings
+
+from django.contrib.auth.views import LoginView
+from django.test import TestCase
+
+from axes.apps import patch_login_view
+
+
+class PatchLoginViewTest(TestCase):
```

Comment:
Use [override_settings()](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.override_settings) to use the newer login view logic.